cognitive/src/main/scala/com/microsoft/azure/synapse/ml/cognitive/search/AzureSearch.scala

```diff
@@ -224,11 +235,12 @@ object AzureSearchWriter extends IndexParser with SLogging {
     val batchSize = options.getOrElse("batchSize", "100").toInt
     val fatalErrors = options.getOrElse("fatalErrors", "true").toBoolean
     val filterNulls = options.getOrElse("filterNulls", "false").toBoolean
+    val vectorColsOpt = options.get("vectorCols")
```

Comment:
nit: in scala the convention isnt usually to name things after the type unless you need to for disambiguation. The idea behind this convention is that the IDE will tell you the type so people prefer to err on the side of sounding more like english